core/server/config/overrides.json

```diff
@@ -67,12 +67,14 @@
     },
     "api": {
         "versions": {
-            "active": {
+            "active": "v2",
+            "stable": "",
+            "deprecated": "v0.1",
```

Comment:
Reason for direct mapping like this was to avoid duplication of config in 2 places(direct version and active), plus its cleaner logically as `active -> v2` and `deprecated -> v0.1`, and each version directly has its config mapped to it.